config/jsons/github-events.json

```diff
@@ -113,10 +113,10 @@
   },
   "star": {
     "created": true,
-    "deleted": false
+    "deleted": true
   },
   "watch": {
-    "started": true
+    "started": false
```

Comment:
@kenautomus Why do we need to change the condition of the watch event?

Reply:
@tanhongit  the started action in this event is also to send the star notification.  

I realized that when there is a new star, there will be 2 events sent at the same time: watch with start action and star with create action

that is the cause of duplication

Reply:
@kenautomus I understood, thanks!